src/components/SingleProduct.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import SearchBar from "./SearchBar";
+import Nav from "./Nav";
+import addToCart from "../redux-store/store";
+import connect from "react-redux";
+
+const SingleProduct = props => {
+  return (
+    <div>
+      <SearchBar />
+      <Nav />
+      {props.product}
```

Comment:
You can de-structure your `props` with `const { product: { name, description, imageUrl } } = props`.